pytest-slc/exasol/pytest_slc/__init__.py

```diff
@@ -33,10 +33,11 @@ def slc_builder():
 @pytest.fixture(scope='session', autouse=True)
 def export_slc_async(slc_builder, use_onprem: bool, use_saas: bool):
     """
-    The fixture starts the export() function of the provided LanguageContainerBuilder object as
-    an asynchronous task.
+    The fixture starts the export() function of the provided
+    LanguageContainerBuilder object as an asynchronous task.
 
-    The operation will be skipped if none of the backends is in use.
+    The operation will be skipped if none of the backends is in use **or the
```

Comment:
"**" is not needed here. This was used as the marking in the ticket description just to highlight the missing text.